addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14448,13 +14458,13 @@ msgstr ""
 #. Description of settings category "Videos -> File lists" with label #14081
 #: system/settings/settings.xml
 msgctxt "#36176"
-msgid "Category containing settings for how video file lists are handled."
+msgid "This category contains the settings for how video file lists are handled."
 msgstr ""
 
 #. Description of setting "Videos -> File lists -> Default select action" with label #22079
 #: system/settings/settings.xml
 msgctxt "#36177"
-msgid "Toggle between Choose, Play (default), Resume and Show Information. Choose will select an item, e.g. open a directory in files mode. Resume will automatically resume videos from the last position that you were viewing them, even after restarting the system."
+msgid "Toggle between \"Choose\", \"Play\" (default), \"Resume\" and \"Show information\". \"Choose\" will select an item (e.g. open a directory in files mode.) \"Resume\" will automatically resume videos from the last position that you were viewing them, even after restarting the system."
```

Comment:
somehow my other comment vanished too early, so here it is again:
don't we use square braces to reference to settings values? We at least did in audio settings. Also I think this description is utterly wrong (choose action will show a context menu):

```
Select the action to perform when selecting a playable item. [CR][Choose] Will ask you each time which of these actions to perform. This is useful for remotes that can not trigger the context menu to see these options. [CR][Play] (default) Will start playback of the video. In case the video has been partly watched, it will ask if you want to resume or start playback from beginning. [CR][Resume] Will also start playback, but automatically resume videos from the last position that you were viewing them instead of asking. [CR][Show Information] Will open a window with detailed information about the video (if available).
```

suggestions @jjd-uk ?